README.md

```diff
@@ -34,12 +34,28 @@ To learn about the steps involved in building from source, see [CONTRIBUTING](./
 
 ## Usage
 
+In order to use the Pinecone CLI you will need to authenticate with Pinecone services. This can be done either with an API key, or using the `pinecone login` flow to authenticate with a Pinecone account and your browser and leverage an authentication token.
+
```

Comment:
This last sentence is a bit long. Maybe just say "to authenticate via your Pinecone account and the browser"?